spezi-module/onboarding/src/main/java/edu/stanford/spezikt/spezi_module/onboarding/invitation/InvitationCodeScreen.kt

```diff
@@ -0,0 +1,88 @@
+package edu.stanford.spezikt.spezi_module.onboarding.invitation
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Edit
+import androidx.compose.material3.Button
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.hilt.navigation.compose.hiltViewModel
+import edu.stanford.spezikt.core.design.component.SpeziValidatedOutlinedTextField
+import edu.stanford.spezikt.core.design.theme.Colors.onPrimary
+import edu.stanford.spezikt.core.design.theme.Colors.primary
+import edu.stanford.spezikt.core.design.theme.Sizes
+import edu.stanford.spezikt.core.design.theme.Spacings
+import edu.stanford.spezikt.core.design.theme.TextStyles.titleLarge
+
+@Composable
+fun InvitationCodeScreen(
+) {
+    val viewModel = hiltViewModel<InvitationCodeViewModel>()
+    val uiState by viewModel.uiState.collectAsState()
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(Spacings.medium),
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        Text(
+            text = "Invitation Code",
+            style = titleLarge,
+        )
+        Spacer(modifier = Modifier.height(Spacings.medium))
```

Comment:
Should we introduce a `VerticalSpacer` composable?

```kotlin
@Composable
fun VerticalSpacer(height: Dp) {
    Spacer(modifier = Modifier.height(height))
}
```
Another alternative: We can already apply `verticalArrangement = Arrangement.spacedBy(Spacings.medium)` in the main `Column` and avoid the need for each spacer

Reply:
Good Idea, I think it would also make sense to give it a default height?
`fun VerticalSpacer(height: Dp = Spacings.medium)`